README.md

```diff
@@ -150,3 +150,6 @@ registry to update it.
 If the branch is already open for PR, it updates it. Take a look at [this example](https://github.com/singularityhub/registry-org/pull/8)
 for the pull request opened when we updated the previous GitHub syntax to the new yaml syntax. Although this
 doesn't describe the workflow above, it works equivalently in terms of the triggers.
+  
+### Proxy
+Building the docker container requires access to pypi.python.org/pypi. If you are running this action ob a self-hosted runner behind a proxy, you can configure the docker client to [flow proxy info to the container](https://docs.docker.com/network/proxy/#configure-the-docker-client). This action will respect the `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` settings in `~/.docker/config.json`, if set.
```

Comment:
I'm confused why the ~/.docker/config.json (on the host, not in the action container) would be relevant here. It seems like setting the envars would be following this instruction? https://docs.docker.com/network/proxy/#use-environment-variables

Reply:
You are exactly right - I've updated my changes to the readme to reflect the docker documentation

Reply:
A few questions:

- Do you have an example screenshot or link to show me this running (and working) in practice? I'd like to see that without these variables, the runner has issue, and adding them resolves it. 
- I'd also like to see that the action works as expected if these variables aren't set.
- Are there any security implications of setting this, especially if the envars aren't used?